versions.json

```diff
@@ -1,16 +1,78 @@
 {
+  "rubygems": "3.5.22",
   "gems": {
     "default": {
+      "abbrev": "0.1.2",
+      "base64": "0.2.0",
+      "benchmark": "0.3.0",
+      "bigdecimal": "3.1.5",
       "bundler": "2.5.22",
+      "cgi": "0.4.1",
+      "csv": "3.2.8",
+      "date": "3.3.4",
+      "delegate": "0.3.1",
       "did_you_mean": "1.6.3",
+      "digest": "3.1.1",
+      "drb": "2.2.0",
+      "english": "0.8.0",
       "erb": "4.0.3",
-      "gem": "3.5.22",
+      "error_highlight": "0.6.0",
+      "etc": "1.4.3",
+      "fcntl": "1.1.0",
+      "fiddle": "1.1.2",
+      "fileutils": "1.7.2",
+      "find": "0.2.0",
+      "forwardable": "1.3.3",
+      "getoptlong": "0.2.1",
       "io-console": "0.7.1",
+      "io-nonblock": "0.3.0",
+      "io-wait": "0.3.1",
+      "ipaddr": "1.2.6",
       "irb": "1.13.1",
+      "json": "2.7.2",
+      "logger": "1.6.0",
+      "mutex_m": "0.2.0",
+      "net-http": "0.4.1",
+      "net-protocol": "0.2.2",
+      "nkf": "0.1.3",
+      "observer": "0.1.2",
+      "open-uri": "0.4.1",
+      "open3": "0.2.1",
+      "openssl": "3.2.0",
+      "optparse": "0.4.0",
+      "ostruct": "0.6.0",
+      "pathname": "0.3.0",
+      "pp": "0.5.0",
+      "prettyprint": "0.2.0",
+      "prism": "0.19.0",
```

Comment:
We actually use a Prism version more recent than that:
https://github.com/truffleruby/truffleruby/blob/774110bc310f5c848c812c308bd47402e1e0f47b/src/main/c/yarp/include/prism/version.h

Probably doesn't matter too much the version is wrong here though.